api/HTMLFencedFrameElement.json

```diff
@@ -151,6 +151,43 @@
           }
         }
       },
+      "sandbox": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/fenced-frame/#element-attrdef-fencedframe-sandbox",
+          "tags": [
+            "web-features:fencedframe"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "117"
```

Comment:
I found https://chromiumdash.appspot.com/commit/2469f173c9941d20dbbe3154cb4f7c51b88285ff, which suggests Chrome 114. 

However, the commit message states:

> As of this CL, the sandbox attribute for fenced frames is only used as a
check **on embedder-initiated navigations**: if the value of the sandbox
attribute doesn't permit the mandatory unsandboxed flags, the navigation
will fail in the renderer.

To me, this suggests that it's only partially implemented in that version?

The [corresponding code comment](https://github.com/chromium/chromium/blob/2efaee0550e093ef3a395e1a7b9dcbbd1598695f/content/browser/fenced_frame/fenced_frame.cc#L354-L358) is still in the sources today.